### PR TITLE
feat: add pix payment

### PR DESCRIPTION
**Pull request by felipe-dna**
- Add pix payment for mercado pago gateway.

### CODE REVIEW

**Comment by jonatasoli** [comment]
Relacionado a issue https://github.com/jonatasoli/fast-ecommerce-front/issues/144

**Review by jonatasoli — DISMISSED** [review]
LGTM!

File: server/api/cart/[uuid]/payment/pix.post.ts.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import { defineEventHandler, getCookie, readBody, createError } from 'h3'
```
Comment (thaismtss) [inline server/api/cart/[uuid]/payment/pix.post.ts.ts:1]: Corrigir o nome do arquivo que tá .ts.ts

**Review by thaismtss — CHANGES_REQUESTED** [review]

File: .idea/workspace.xml [hunk 2]
```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
[the remark this answers is not in the document]
Comment (felipe-dna) [reply .idea/workspace.xml:1]: 👀

**Pull request merged**